### ISSUE TITLE
Instance: The `.chat-thread-toggle` controls do not include their visible text in their accessible names | WCAG 2.2 Violation - There are visible labels present within the interface whose content is not included in the accessible names of […]

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#43]
### Issue

There are controls with visible labels that don't include the text of those visible labels in their accessible names.

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.5.3 Label in Name](https://www.w3.org/TR/WCAG22/#label-in-name)

### User Impact

Some users - such as people using voice control software - may be unable to access, focus, or activate the controls. Screen reader users may be confused by the controls being announced by a different accessible name than their visible labels.

#### Further information:

1. 

### Screenshot

![Screenshot of an example of one of the \`.chat-thread-toggle\` controls in its collapsed and expanded states. In both states, the control's visible text reads '1 reply'. In its collapsed state, the control's computed accessible name is 'Expand replies'. In its expanded state, the control's computed accessible name is 'Collapse replies'.](https://github.com/user-attachments/assets/288ce04b-bad9-48bb-8fad-655247f1b91a)

### Code Reference

```
<!-- An example of one the controls in a collapsed state -->
<button class="chat-thread-toggle" […] aria-expanded="false" aria-label="Expand replies">
	[…]
	<span class="chat-reply-count">1 reply</span>
	[…]
</button>

<!-- The same control in an expanded state -->
<button class="chat-thread-toggle" […] aria-expanded="true" aria-label="Collapse replies">
	[…]
	<span class="chat-reply-count">1 reply</span>
	[…]
</button>
```

#### Using the Chrome developer tools:

1. Take note of the visible text of the control's label.
2. Inspect the control using Chrome's developer tools.
3. Activate the "Accessibility" tab (for the element) within the developer tools.
4. Under "Computed Properties", notice that the "Name" value does not include the visible text of the control's label (noting that _all_ of the visible text of the control's label must be included in the control's accessible name, verbatim).

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader.
2. Navigate to the control.
3.
